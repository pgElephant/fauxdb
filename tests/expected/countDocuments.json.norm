{
  "n": 100,
  "ok": 1
}
