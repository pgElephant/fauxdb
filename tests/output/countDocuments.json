{ ok: 1, n: Long('100') }
