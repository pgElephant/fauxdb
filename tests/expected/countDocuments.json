{
  "ok": 1,
  "n": 100
}
